Makefile

```diff
@@ -13,6 +13,7 @@ SERVER_DIR:=images/server
 AD_SERVER_DIR:=images/ad-server
 CLIENT_DIR:=images/client
 SERVER_SRC_FILE:=$(SERVER_DIR)/Dockerfile.fedora
+SERVER_SOURCES:=$(SERVER_DIR)/setuser.sh $(SERVER_DIR)/smb.conf $(SERVER_DIR)/smb.import.conf
```

Comment:
I don't know if we still use these files. It might be better to remove them if we can? Maybe we left them behind before sambacc was more mature? But I think sambacc should be the way forward (of course I am bound to think that :-))

Reply:
fixed for this one. The current dockerfile only copies `smb.conf`.  We can remove the other files (+their dockerfile) later.